Hello World Program in Dart Language

ShaikSarfraj-tech opened the pull request:
I have written the "Hello, World!" Program in Dart Language, Kindly Accepts it.
Thank You.
[Hello_World in Dart.zip](https://github.com/rpotter12/data-structure/files/5338089/Hello_World.in.Dart.zip)

rpotter12 committed:
Merge pull request #2 from rpotter12/newbranch

changing file location

rpotter12 committed:
circular queue

rpotter12 committed:
updating readme.md

rpotter12 committed:
basic language syntax

rpotter12 committed:
rearrange

rpotter12 committed:
rust syntax

rpotter12 committed:
basic syntax program

rpotter12 committed:
updating readme.md

rpotter12 committed:
aranges files

rpotter12 committed:
Merge pull request #3 from rpotter12/pull

aranges files

rpotter12 committed:
update readme.md

rpotter12 committed:
Merge pull request #4 from rpotter12/pull

update readme.md

rpotter12 committed:
basic syntax

rpotter12 committed:
Merge pull request #5 from rpotter12/pull

basic syntax

rpotter12 committed:
update readme.md

rpotter12 committed:
update readme.md

rpotter12 committed:
insertion at first

rpotter12 committed:
BFS

rpotter12 committed:
bfs and binary search

rpotter12 committed:
binary search

rpotter12 committed:
min max by divide and conquer

rpotter12 committed:
bubble sort

rpotter12 committed:
circular linked list

rpotter12 committed:
Merge pull request #6 from rpotter12/pull

circular linked list

rpotter12 committed:
inorder successor

rpotter12 committed:
Merge pull request #7 from rpotter12/pull

inorder successor

rpotter12 committed:
trees

rpotter12 committed:
linear search

rpotter12 committed:
Merge pull request #8 from rpotter12/pull

trees and search

rpotter12 committed:
kruskals and prims

rpotter12 committed:
Merge pull request #9 from rpotter12/pull

kruskals and prims

rpotter12 committed:
doubly-linked-list

rpotter12 committed:
Merge pull request #10 from rpotter12/pull

doubly-linked-list

rpotter12 committed:
inorder

rpotter12 committed:
Merge pull request #11 from rpotter12/pull

inorder

foxtrot12 committed:
Binary search for C

foxtrot12 committed:
Merge pull request #1 from foxtrot12/foxtrot12-patch-1

Binary search for C

foxtrot12 committed:
Revert "Binary search for C"

foxtrot12 committed:
Python-BinarySearch

foxtrot12 committed:
Merge pull request #2 from foxtrot12/foxtrot12-patch-2-Python-BinarySearch

Python-BinarySearch

foxtrot12 committed:
Python-BinarySearch

rpotter12 committed:
Merge pull request #12 from foxtrot12/revert-1-foxtrot12-patch-1

Revert 1 foxtrot12 patch 1 - Binary Search C

rpotter12 committed:
Merge pull request #13 from foxtrot12/master

Python and C - Binary Search

rpotter12 committed:
arranging in folders

rpotter12 committed:
deletion at the beginning of linked list

rpotter12 committed:
Merge pull request #14 from rpotter12/pull

deletion at first and file arranged

rpotter12 committed:
deletion at last in linkedlist

rpotter12 committed:
file removal

rpotter12 committed:
Merge pull request #15 from rpotter12/pull

deletion at end in linked list

jrcamelo committed:
Added Python Quick Sort

rpotter12 committed:
Merge pull request #16 from jrcamelo/master

Added Python Quick Sort

rpotter12 committed:
merge sort

rpotter12 committed:
Merge branch 'master' of https://github.com/rpotter12/data-structure into pull

rpotter12 committed:
Merge pull request #17 from rpotter12/pull

merge sort

JainStuti25 committed:
Create heapSort.py

rpotter12 committed:
Merge pull request #18 from JainStuti25/master

Create heapSort

rpotter12 committed:
stack using array

rpotter12 committed:
Merge pull request #19 from rpotter12/pull

stack using array

vaibhavomer committed:
Queue using array

rpotter12 committed:
Merge pull request #20 from vaibhavomer/master

Queue using array

vaibhavomer committed:
DFS

rpotter12 committed:
DFS

rpotter12 committed:
Merge pull request #21 from vaibhavomer/master

DFS

vaibhavomer committed:
Circular_linked_list

vaibhavomer committed:
Merge remote-tracking branch 'upstream/master'

rpotter12 committed:
Merge pull request #22 from vaibhavomer/master

circular doubly linkedlist

AtoMc committed:
Create BinarySearchRecursive.java

rpotter12 committed:
Merge pull request #23 from AtoMc/master

Create BinarySearchRecursive.java

vaibhavomer committed:
queue using stack

vaibhavomer committed:
Merge remote-tracking branch 'upstream/master'

rpotter12 committed:
Merge pull request #24 from vaibhavomer/master

implement queue using stack

rpotter12 committed:
stack using linked list

rpotter12 committed:
Merge pull request #25 from rpotter12/pull

stack using linked list

rpotter12 committed:
queue using linkedlist

rpotter12 committed:
Merge pull request #26 from rpotter12/pull

queue using linkedlist

rpotter12 committed:
insertion in binary tree

rpotter12 committed:
Merge pull request #27 from rpotter12/pull

insertion in binary tree

rpotter12 committed:
insertion sort

rpotter12 committed:
Merge pull request #28 from rpotter12/pull

insertion sort

rpotter12 committed:
selection sort

rpotter12 committed:
Merge pull request #29 from rpotter12/pull

selection sort

rpotter12 committed:
updated readme.md

rpotter12 committed:
Merge pull request #30 from rpotter12/pull

updated readme.md

Commit:
Binary_index_tree Algo Added :)

namishkhanna committed:
Update README.md

namishkhanna committed:
Create hello-world in php

rpotter12 committed:
Merge pull request #32 from namishkhanna/master

Update README.md

rpotter12 committed:
Merge pull request #31 from JD235/prog

Binary_index_tree Algo Added :)

kushasharma committed:
bubble sort

rpotter12 committed:
Merge pull request #34 from kushasharma/master

bubble sort

rpotter12 committed:
Merge pull request #35 from Gauri0799/master

Tree Implementation

Gauri0799 committed:
Tree Implementation

rpotter12 committed:
deleted hmm.cpp file

rpotter12 committed:
Merge pull request #36 from rpotter12/pull

deleted hmm.cpp file

Harnoor6693 committed:
Update README.md

rpotter12 committed:
Merge pull request #37 from Harnoor6693/patch-1

Update README.md

rpotter12 committed:
doubly-linked-list-deletion

rpotter12 committed:
Merge branch 'master' of https://github.com/rpotter12/data-structure into pull

rpotter12 committed:
Merge pull request #38 from rpotter12/pull

doubly linked list deletion

rpotter12 committed:
insertion sort

rpotter12 committed:
Merge pull request #40 from rpotter12/pull

insertion sort

rpotter12 committed:
merge sort

rpotter12 committed:
Merge pull request #41 from rpotter12/pull

merge sort

rpotter12 committed:
Delete prims.cpp

rpotter12 committed:
Delete kruskals.cpp

rpotter12 committed:
heap sort

rpotter12 committed:
Merge pull request #42 from rpotter12/pull

heap sort

rpotter12 committed:
selection sort

rpotter12 committed:
Merge pull request #43 from rpotter12/pull

selection sort

rpotter12 committed:
quick sort

rpotter12 committed:
Merge pull request #44 from rpotter12/pull

quick sort

rpotter12 committed:
infix to postfix

rpotter12 committed:
Merge pull request #45 from rpotter12/pull

infix to postfix

rpotter12 committed:
array

rpotter12 committed:
Merge pull request #46 from rpotter12/pull

array

rpotter12 committed:
Delete minmax.cpp

rpotter12 committed:
minmax

rpotter12 committed:
Merge pull request #47 from rpotter12/pull

minmax

rpotter12 committed:
deleted extra file

rpotter12 committed:
Merge pull request #48 from rpotter12/pull

deleted extra file

rpotter12 committed:
rename php file

rpotter12 committed:
Merge pull request #49 from rpotter12/pull

rename php file

rpotter12 committed:
sounting sort

rpotter12 committed:
removed unneccessary files

rpotter12 committed:
Merge pull request #50 from rpotter12/pull

counting sort

rpotter12 committed:
counting sort python

rpotter12 committed:
Merge pull request #51 from rpotter12/pull

counting sort python

rpotter12 committed:
counting sort java

rpotter12 committed:
Merge pull request #52 from rpotter12/pull

counting sort java

rpotter12 committed:
2-d array

rpotter12 committed:
arranging files

rpotter12 committed:
matrix multiplication

rpotter12 committed:
upper and lower traingle

rpotter12 committed:
matrix addition

rpotter12 committed:
array rotation

rpotter12 committed:
hello world in bash

rpotter12 committed:
Update Readme to interview preperation

rpotter12 committed:
Merge pull request #60 from rpotter12/pull

Update README.md to interview preperation

rpotter12 committed:
warm-up round and pattern preperation

rpotter12 committed:
Merge pull request #61 from rpotter12/pull

Add 2 rounds

namishkhanna committed:
warm-up round and pattern preperation

Print Hello World in C# (C Sharp)

namishkhanna committed:
warm-up round and pattern preperation

Printing Hello World in Kotlin

rpotter12 committed:
Merge pull request #64 from namishkhanna/patch-2

Hello World in Kotlin

rpotter12 committed:
Merge pull request #62 from namishkhanna/patch-1

warm-up round and pattern preperation

rpotter12 commented:
@ShaikSarfraj-tech You are messed up with your PR. Please use this following command:
```
$ git remote add upstream https://github.com/rpotter12/data-structure
$ git fetch upstream master
$ git reset --hard upstream/master
// Do the changes
$ git push -f origin master
```
And make pull request. And you are generating PR for master branch only.

ShaikSarfraj-tech commented:
Sir, It is Showing like this
remote: Permission to rpotter12/data-structure.git denied to ShaikSarfraj-tech.
I don't understand what should I do now.
Please help me.

rpotter12 committed:
recursion questions

rpotter12 committed:
Merge pull request #66 from rpotter12/pull

recursion questions

rpotter12 committed:
recursion questions

rpotter12 committed:
Merge pull request #67 from rpotter12/pull

recursion questions

Niharika248 committed:
new question 2

Niharika248 committed:
input for question 2

Niharika248 committed:
Delete input2.txt

Niharika248 committed:
solution code for question 2

Niharika248 committed:
matrix multiplication

rpotter12 committed:
Merge pull request #68 from Niharika248/master

new question 2

rpotter12 committed:
Dynamic Programming questions

rpotter12 committed:
Merge pull request #69 from rpotter12/pull

Dynamic Programming questions

rpotter12 committed:
Searching And Sorting

rpotter12 committed:
Merge pull request #70 from rpotter12/pull

Searching And Sorting

ShaikSarfraj-tech committed:
Added Hello World Program in Dart Language

Niharika248 committed:
Lucas Sequence of Numbers Printing 

The Lucas numbers or series are an integer sequence named after the mathematician François Édouard Anatole Lucas, who studied both that sequence and the closely related Fibonacci numbers. Lucas numbers and Fibonacci numbers form complementary instances of Lucas sequences.
The sequence of Lucas numbers is: 2, 1, 3, 4, 7, 11, 18, 29, ….

Niharika248 committed:
Printing Pattern in Java

 Diamond Shape Pattern Program in Java
Enter the number of rows: 5

rpotter12 committed:
Merge pull request #72 from Niharika248/master

Lucas Sequence of Numbers Printing

rpotter12 committed:
Merge pull request #71 from ShaikSarfraj-tech/hello

Added Hello World Program in Dart Language

Niharika248 committed:
Recursive Factor Program 

program for recurrence relation to find the Factorial of n where n>2 .

rpotter12 committed:
Merge pull request #73 from Niharika248/master

Recursive Factor Program

Niharika248 committed:
Addition of 2 matrices in java

We can add two matrices in java using binary + operator. A matrix is also known as array of arrays. We can add, subtract and multiply matrices.

rpotter12 committed:
Merge pull request #74 from Niharika248/master

Addition of 2 matrices in java

Niharika248 committed:
Create Finding All SCS

Finding Shortest Common Super Sequence (Z) of given sequences X and Y such that both X and Y are subsequences of Z.

Niharika248 committed:
Create Recursive Func for Prime

Given a Number, Recursive Function for a Prime Number

rpotter12 committed:
Merge pull request #75 from Niharika248/master

Create Finding All SCS

Niharika248 committed:
create a program for markov matrix

Niharika248 committed:
Create Pattern 7

rpotter12 committed:
Merge pull request #76 from Niharika248/master

create a program for markov matrix

Niharika248 committed:
Create Matrix 3

rpotter12 committed:
Merge pull request #77 from Niharika248/master

Create Matrix 3

Niharika248 committed:
Reverse a stack Using Recursion

rpotter12 committed:
Merge pull request #78 from Niharika248/master

Reverse a stack Using Recursion
